src/earn/incentives.md

```diff
@@ -4,41 +4,67 @@ title: Project Artemis
+#### Validator Tasks
 
-**Phase 2 - Buzz: Platform Testing**
+|**Task ID**|**Description**|**Resources**|**Reward**|
+| - | - | - | - |
+|`ARTEMIS-APOLLO#1`|Setup a Cudos Validator|[Docs](/build/#how-to-run-a-cudos-validator-node)|10|
+|`ARTEMIS-APOLLO#2`|Add a Sentry Node to your Validator|[Docs](/build/#run-a-sentry-node)|5|
+|`ARTEMIS-APOLLO#3`|Add Prometheus monitoring to the Validator Node|[Docs](https://prometheus.io/docs/prometheus/latest/getting_started/)|-|
+|`ARTEMIS-APOLLO#4`|Create a Grafana dashboard to show `ARTEMIS-APOLLO#3` Prometheus data/sending Validator data monthly to Cudo Network|[Docs](https://prometheus.io/docs/visualization/grafana/)|100|
+|`ARTEMIS-APOLLO#6`|Write a unique blog post setting up a Validator Node (in any language of your choice)|n/a|-|
+
+#### Developer Tasks
+
+|**Task ID**|**Description**|**Resources**|**Reward**|
+| - | - | - | - |
+|`ARTEMIS-APOLLO#5`|Install Keplr and get some tokens from the faucet using the explorer|[Docs](/build/account-setup.html)|10|
+|`ARTEMIS-APOLLO#7`|Join our community [Telegram](https://t.me/cudostelegram), [Discord](https://discord.com/invite/NUmUXEmvBn), and follow CUDOS on [Twitter](https://twitter.com/CUDOS_)|n/a|1|
+
+### Phase 2 - Buzz (Launching Soon)
 
 Test all aspects of Validation, including staking, delegated staking, rewards. Additionally exploring CosmWasm smart contracts including: contract interaction, contract deployment, contract interfaces. Testing of our Gravity bridge implementation with Rinkeby Ethereum.
 
-**Phase 3 - Armstrong: Adversarial Stress Test**
+|**Task ID**|**Description**|**Resources**|**Reward**|
+| - | - | - | - |
+|`ARTEMIS-BUZZ#1`| 🌎 | 🚀 | 🌝 |
+
+### Phase 3 - Armstrong (Launching Soon)
 
-A coordinated strike to bring as much diversity in transaction activity as possible to the Cudos Network. Validators will run network load and performing maintenance tasks.
+Adversarial stress tests bringing a coordinated strike to bring as much diversity in transaction activity as possible to the Cudos Network. Validators will run network load and performing maintenance tasks.
 
-**Phase 4 - ?: Migration Testing**
+|**Task ID**|**Description**|**Resources**|**Reward**|
+| - | - | - | - |
+|`ARTEMIS-ARMSTRONG#1`| 🌎 | 🚀 | 🌝 |
 
-Dry-run of our genesis and migration plan for the new native Cudos Network
+### Phase 4 - Collins (Launching Soon)
+
+Migration testing to performa a dry-run of our genesis and token migration plan for the new native Cudos Network
+
+|**Task ID**|**Description**|**Resources**|**Reward**|
+| - | - | - | - |
+|`ARTEMIS-COLLINS#1`| 🌎 | 🚀 | 🌝 |
 
 ## How to Participate
 
-The signup form for brave Cudo-naughts is now open and can be found here, we need a few pieces of info to ensure we can distribute tokens in the airdrop:
+Take a look at the [active tasks](#tasks) for the current phase and dive into the Somniorum Testnet
```

Comment:
the tasks section has been removed, so this doesn't lead anywhere at the moment. Should we add it back?

Reply:
good spot!